### ISSUE TITLE
Functionality of the screen is operable through Tab and arrow kays but there is no such indication, Design issue.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#804]
**Describe the bug**
Functionality of the screen is operable through Tab and arrow kays but there is no such indication, Design issue.

**To Reproduce**

1. Go to Design  (http://localhost:9992/cm/app/?view=design&site=test4&mode=readonly&name=index.html&path=%2FSites%2Ftest4%2Findex.html&pathType=page)
2. Create a site and click on it.
3. There are Radio buttons and File Choose Option under Add template and Import template Pop Up, under Action menu respectively. 4. These are accessible using Tab and Arrow key but not indicated.

**Expected behavior**
if Functionality of the screen is operable through Tab and arrow kays then indication should be there.

**Additional context**
Navigate all the options available for similar issue and fix.
